### ISSUE TITLE
Portable mode is failing

### ISSUE THREAD

**Issue by Danixu** [vboxme/Portable-VirtualBox#34]
When you change the program folder or even just the drive letter, the machines are not updated and stop working. You can edit the VirtualBox.xml file manually and make it work, but then is not fully portable.

Maybe the problem is because i've installed VirtualBox on my computer, but where's the difference?.

Greetings and thanks for this nice program!.

**Comment by p060477** [comment]
does bridge network work for you?
but are you able to bi directional drag and drop from host to guest and vice-versa..??
have you win 10 as host?

**Comment by Danixu** [comment]
Yes, i'm using Windows 10 as host, bridge mode is working and drag and drop is not working. Anyway, i've installed the VirtualBox on my computer...

**Comment by p060477** [comment]
hi and thxs for yr reply!
but bridge network worked ALSO when you have vbox PORTABLE..??
i mean with win 10 as host
or is only working NOW that you have INSTALLED it..??
this is the place i think for poeple who have issues with the portable version..
otherwise you have the official forum where they are so happy to assist you
but really not happy at all to assist poeple,like us here, who have choosen NOT to install it..

**Comment by Danixu** [comment]
Thanks, but i've posted here because I've problems with portable version of VirtualBox. The Installed version works near to perfect (don't works drag and drop but all the other things works fine).
My problem with the portable version is that when you change the path or the letter of the portable folder, the paths on the VirtualBox.xml are not updated and then the Virtual Machines fails. If i update the file manually then it works.
Maybe is a problem with W10, because i've tested the portable on a W7 guest machine and seems to work. I've to test it again on other computer.

About the bridge, i don't know if works without install virtualbox because i had it installed before try the portable. 
i'm trying to create a portable version to use it on other computers without install the whole program.
If you wanna know it, i can uninstall the Vbox and try if works.

Greetings!!

**Comment by Megachip** [comment]
@Danixu that shouldn't happen. Did you try to double restart pvb on the drive with the new letter? Have the experience that sometime on the first run on the new letter something "hangs".

Can't remember, that it did not work if vb is already installed. But I think it is generally not recommended to use pvb and installed vb on the same machine. Whats your use case for that?

**Comment by p060477** [comment]
hi danixu!
first thkxs so much for yr proof!
its very strange that drag n drop won't' work even with installed vb..
anyway i'd really like to know if the bridge network works for you ALSO with the portable vbox
it should be very very meaningfull
cheers!

**Comment by runarbu** [comment]
If you already have VirtualBox installed, and run Portable-VirtualBox then Portable-VirtualBox will use the existing VirtualBox installation. Having both Portable-VirtualBox  and VirtualBox on the same machine is not very common, so there may be some bugs that only effect that setup, that we have missed.

@Danixu: Can you post the xml keys you have to edit?

**Comment by p060477** [comment]
your english is perfect
and also yr proof!
the matter is with win 10 as host..are you able to use drag and drop with win 10 as host..??
and moreover are you able to use bridge networking..??
-that's the question- (..also shakespeare said..!..;)..)

**Comment by p060477** [comment]
another question..:
which is the right way to remove completely the vbox -portable- vers from the pc..??
cause so many reg keys are involved even if it is a portable app..

**Comment by Danixu** [comment]
Sorry for the delay, i was bussy this week.

I've uninstalled VirtualBox and i've seen that with portable version the bridge is not working (at least on Windows 10 the network adapter is not installed). Drag and drop fails on portable, and i've installed the latest version and is working fine (My portable is using the latest too).

The portable now is working as is expected (the paths are updated and VM are working). I don't know why that day was failing all...

The only way to clean the whole virtualbox from the computer after close it is deleting all the installed registry keys and removing the drivers. Maybe the portable is doing all already.

Greetings!!

**Comment by p060477** [comment]
Hi Danixu!
first thxs again!
..you are the one here that do not -completely- disappear..so thxs again!
so you confirm that with win 10 as host and vbox portable:
1) drag n drop does not work bi-directional so also from host to guest
2) the bridge network DOES NOT work still..
the great pity is that on the main official page of vbox.me there are the instructions to have it
working..and are there since at least the issue of win 10..from which that feauture really
DOES NOT work AT ALL...what a SHAME..
thxs so much for yr kind and precious proof!
we are so only all anxiously waiting for a final PATCH
cheers!!

**Comment by p060477** [comment]
hi there!
..why you are disappeared again..??
why to have this sw portable is this long nightmare..??
why on the project page vbox.me is still clearly written how to have network bridged if since JULY 2015 IT IS CLEARLY - IMPOSSIBLE-..??
RUNAR..?
MEGACHIP..?
where are you..???

**Comment by ju-wien** [comment]
>>
If you already have VirtualBox installed, and run Portable-VirtualBox then Portable-VirtualBox will use the existing VirtualBox installation. Having both Portable-VirtualBox and VirtualBox on the same machine is not very common, so there may be some bugs that only effect that setup, that we have missed.
<<

I think this is a very important information that should be included in the instructions for installing Portable-VirtualBox. I came here because my installation process looked a lot different from what is described at http://www.vbox.me/. The installer only let me choose the destination folder for extracting the files, nothing else and it did not downlad anything either (or it was *very* fast). Your remark makes this understandable. (If you update the instructions you could also remove the outdated links to win-lite.de.)

The reason why I have VirtualBox installed and want a portable version on my USB-drive, too is, because I've been using VirtualBox for years and I also want to use it elsewhere without carrying my computer along (and without installing anything on computers that are not my own).

**Comment by p060477** [comment]
oh my poor ju-wien..!..my brother in this very frustrating NIGHTMARE..
all here seem to be DISAPPEARED
MEGACHIP..
RUNAR..
all they went away without any REPLY
what a PITY
what a SHAME
